src/main/java/net/fabricmc/api/ClientModInitializer.java

```diff
@@ -16,7 +16,29 @@
 
 package net.fabricmc.api;
 
+/**
+ * A mod initializer ran only on {@link EnvType#CLIENT}.
+ *
+ * <p>This entrypoint is suitable for setting up client-specific logic, such as rendering
+ * or integrated server tweaks.</p>
+ *
+ * <p>In {@code fabric.mod.json}, the entrypoint is defined as:
+ * <pre><blockquote>
+ *     "entrypoints": {
+ *         "client": [
+ *             &lt;a list of string referring to your client mod initializers&gt;
```

Comment:
This might not be the right place for this, as it can be diffrent for diffrent langs

Reply:
should make it `entrypoint list notation` instead, I guess?

Reply:
People can just look at the docs on getEntrypointContainers, I think.